src/interface/src/app/plan/create-scenarios/goal-overlay/goal-overlay.component.html

```diff
@@ -0,0 +1,20 @@
+<mat-card *ngIf="goal$ | async as goal">
+  <header>
+    <h4>{{ goal?.short_question_text }}</h4>
+    <button mat-icon-button (click)="close()" aria-label="close">
+      <mat-icon class="close-btn">close</mat-icon>
+    </button>
+  </header>
+  <section>
+    <p>{{ goal?.long_question_text }}</p>
+    <p *ngFor="let p of goal?.description" [innerHTML]="p"></p>
+    <p>
+      For more information, see the
+      <a
+        href="https://www.planscape.org/documentation/user-guide/"
+        target="_blank"
+        >User Guide</a
```

Comment:
Will all of the links point to the same place for each goal, or are there going to be individual links for them at some point?

Reply:
the one on the doc shared is the same for all

Reply:
also the guide does not have anchors/links to individual sections :( (might be worth adding!)